types/Appointment.ts

```diff
@@ -1,7 +1,11 @@
+import { Patient } from './Patient';
+
 export type Appointment = {
     id: string,
     psychologistId: string,
     appointmentDate: string,
     patientId: string,
     deleted: boolean
 }
+
+export type AppointmentWithPatient = Appointment & Patient;
```

Comment:
:thinking: shouldn't this done on the specific place where it's used? It's not a 'real' type.